src/pocket/PocketEventBridgeWithLambdaTarget.ts

```diff
@@ -27,8 +30,11 @@ export class PocketEventBridgeWithLambdaTarget extends PocketVersionedLambda {
   ) {
     super(scope, name, config);
 
-    const eventBridgeRule = this.createEventBridgeRule(this.lambda);
-    this.createLambdaEventRuleResourcePermission(this.lambda, eventBridgeRule);
+    const eventBridgeRule = this.createEventBridgeRule([this.lambda]);
+    this.createLambdaEventRuleResourcePermission(
+      [this.lambda],
```

Comment:
since the class extends `PocketVersionedLambda`, couldn't modify this class to get multiple lambda. so retained the functionality of this class as it is, and rolled out new pocket construct.

this way, we also wont have this as a breaking change